Microsoft.ML.FairLearn/api-example.cs

```diff
@@ -0,0 +1,52 @@
+// See https://aka.ms/new-console-template for more information
+using Microsoft.ML;
+using Microsoft.ML.AutoML;
+using Microsoft.ML.FairLearn;
+
+Console.WriteLine("Hello, World!");
+
+IDataView testData = null;
+var context = new MLContext();
+
+// train a model using fair learn
+// create a fairlearn experiment using AutoML
+var experiment = context.Auto().CreateExperiment();
+
+// reducing demographic parity using "Zone" as sensitive feature.
+// 0.9: differenceBound
+// 0.5: constraint weight
+// 10: grid limit
+// 100: grid size
+// "Zone": Sensitive feature
+experiment.ReduceDemographicParityUsingGridSearch(0.9, 0.5, 10, 100, "Zone");
+
+// or
+var constraint = new DemographicParity(bound: 0.9);
```

Comment:
`bound` is a bit general. Do you mean a difference or ratio bound? See https://github.com/fairlearn/fairlearn/blob/a481cff8313b7c02a837d544adba121dd5de37e8/fairlearn/reductions/_moments/utility_parity.py#L99

I've written about this a bit on the user guide, too: https://fairlearn.org/v0.7.0/user_guide/mitigation.html#reductions